packages/eslint-plugin-gestalt/src/validators.js

```diff
@@ -109,14 +109,47 @@ export const validateBorder = (value: string): ?string => {
     cleanValue === '#eee 1px solid' ||
     cleanValue === '1px lightgray solid'
   ) {
-    return '  Use prop `borderSize="sm"` instead';
+    return '  Use prop `borderStyle="sm"` instead';
   }
   if (
     cleanValue === '#efefef 2px solid' ||
     cleanValue === '#eee 2px solid' ||
     cleanValue === '2px lightgray solid'
   ) {
-    return '  Use prop `borderSize="lg"` instead';
+    return '  Use prop `borderStyle="lg"` instead';
+  }
+  return undefined;
+};
+
+export const validateBoxShadow = (value: string): ?string => {
+  // If the value is a string:
+  // 1) strip out the rgba portion
+  // 2) convert the pixel portion to only numbers
+  // 3) If both pieces match, recommend borderStyle="shadow"
+
+  const rgbaRegex =
+    '/rgba\\(\\s*(-?\\d+|-?\\d*\\.\\d+(?=%))(%?)\\s*,\\s*(-?\\d+|-?\\d*\\.\\d+(?=%))(\\2)\\s*,\\s*(-?\\d+|-?\\d*\\.\\d+(?=%))(\\2)\\s*,\\s*(-?\\d+|-?\\d*.\\d+)\\s*\\)/g';
```

Comment:
:dragon-faint:

Reply:
Yeah I definitely did not make this 😂 thank you internet